src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,12 @@
-export const CommentList = () => (
-  <>Put the list here</>
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
+
+export const CommentList = ({ comments }) => (
+  <>
+    <div className="CommentList">
+      {comments.map(comment => (
```

Comment:
You should handle the case when comments is undefined or empty. This will prevent potential errors during rendering.